service/dap/server_test.go

```diff
@@ -1679,6 +1680,7 @@ func TestGlobalScopeAndVariables(t *testing.T) {
 					// Step into pkg.AnotherMethod()
 					client.StepInRequest(1)
 					client.ExpectStepInResponse(t)
+					client.ExpectContinuedEvent(t)
```

Comment:
Wouldn't it be better to change the code generator so that ExpectStepInResponse etc also always check for a continued event?

Reply:
I think it is helpful to have explicit sequences in the tests. They serve as documentation of sorts.

Reply:
If you insist. I think it's very repetitive. And besides, the specific sequence is an implementation quirk, it would still be compliant if they came in in the other order.

Reply:
I am hoping they will respond to my feature request and we wouldn't need any of them.

Reply:
I'm fine with the extra verbosity for now, but I agree with @aarzilli that this could be tedious / potentially fragile (testing the implementation rather than the behavior).

Reply:
I think I will make the change to the code generator, since this will be easier to remove if the dap change gets implemented and we can someday stop sending these events, instead of finding all the continuedevents that need to be removed.

Reply:
I can do that in a different PR since its more of a code health issue.